dashboard/app/models/levels/dsl_defined.rb

```diff
@@ -34,6 +34,16 @@ def dsl_default
     "Enter the level definition here.\n"
   end
 
+  def localized_text(text)
+    I18n.t(
+      text,
+      scope: ['data', type.underscore, name],
+      separator: I18n::Backend::Flatten::SEPARATOR_ESCAPE_CHAR,
+      default: text,
+      smart: true
+    )
+  end
```

Comment:
Note that this method here is the original goal of this work; "create a set of standard methods for extracting strings from DSL files". This is said method.

Reply:
Note also that it can easily be overridden by any of the (many, many) classes that extend from this one, to provide per-level-type control over our i18n strategy.